activiti-services/activiti-services-connectors/src/main/java/org/activiti/services/connectors/model/ServiceTaskEvent.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 Alfresco, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.activiti.services.connectors.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ServiceTaskEvent {
```

Comment:
@erdemedeiros not sure about this structure. I think that we need just one structure. IntegrationEvent which will contain the content of the AsyncContext plus the correlationId plus the variables. is there a reason to have all those types ?

Reply:
Well, the SeviceTaskEvent is going to be renamed to IntegrationEvent. I initially grouped executionId, processInstanceId and taskId inside AsyncContext because they are all related to the engine core execution and this avoid having an ugly constructor with too many parameters.
I'm not sure that taskId is actually necessary and we can rename AsyncContext to something else.